### PR TITLE
Footer change + starting sections

### PR DESCRIPTION
**Pull request by anitamalina**
Changed the footer to be different on each screen, so we can add different buttons on each screen.

### CODE REVIEW

**Comment by anitamalina** [comment]
I'm still missing doing it on the dashboard screen - because I need to see Helena's changes ;)

File: src/components/buttons/AddEmployeeButton/AddEmployeeButton.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const AddEmployeeButton = ({
+    text
+}) => {
+
+    function handleClickPopup() {
+
+    }
+
+    return (
+        <button className="addemployee-iconbtn" onClick={handleClickPopup}  >
```
Comment (helher) [inline src/components/buttons/AddEmployeeButton/AddEmployeeButton.js:14]: It is a small preference (and in no way necessary), but could our "own" class names be divided based on words? Such that `addemployee-iconbtn` becomes `add-employee-icon-btn`

File: src/components/buttons/AddEmployeeButton/AddEmployeeButton.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './AddEmployeeButton.css';
+import { IoIosAddCircleOutline } from 'react-icons/io'
+
+const AddEmployeeButton = ({
+    text
+}) => {
+
+    function handleClickPopup() {
```
Comment (helher) [inline src/components/buttons/AddEmployeeButton/AddEmployeeButton.js:9]: I am not quite sure I know what this one is responsible for. Can you elaborate? 🍏

File: src/components/buttons/AddIdeaButton/AddIdeaButton.js [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
     text,
-    UserIcon,
-    trigger,
+    setPopup,
     setIdeaId
 }) => {
 
     function handleClickPopup() {
-        trigger(true)
+        console.log("bt clicked!")
```
Comment (helher) [inline src/components/buttons/AddIdeaButton/AddIdeaButton.js:13]: I assume you have used `console.log("bt clicked!")` for debugging. I do the exact same, but before we hand in the code/exam, we should probably just remember to remove them 😅

File: src/components/buttons/AddIdeaButton/AddIdeaButton.js [hunk 4]
```diff
@@ -27,7 +28,7 @@ const AddIdeaButton = ({
             setIdeaId(ididea)
             console.log("ideaid", ididea)
             console.log("idea in brackets: ", (createIdeaInDB = {ididea}))
-/*             alert("Idea with id: " + ididea + " is created - HURRRA!") */
+/*          alert("Idea with id: " + ididea + " is created - HURRRA!") */
```
Comment (helher) [inline src/components/buttons/AddIdeaButton/AddIdeaButton.js:31]: Is it possible to style `alert`s? I think that is possible, but we have not considered it in our Figma design, right? 🧐

File: src/components/dropdowns/DropdownCalendar/DropdownCalendar.js [hunk 5]
```diff
@@ -1,37 +1,36 @@
...
-            value={expirationDateSelected}
-            onChange={setExpirationDateSelected}
+            value={expirationDate}
+            onChange={setExpirationDate}
             inputPlaceholder="Select" // placeholder
             formatInputText={formatInputValue} // format value
             inputClassName="datePicker" // custom class
         />
-            <IoMdArrowDropdownCircle className="dropdowncalender-icon"/>
+            {/* <IoMdArrowDropdownCircle className="dropdowncalender-icon"/> */}
```
Comment (helher) [inline src/components/dropdowns/DropdownCalendar/DropdownCalendar.js:33]: Is it meant to be commented?

File: src/components/footer/Footer.js [hunk 6]
```diff
@@ -1,17 +1,13 @@
...
-function Footer(props) {
+function Footer() {
     
     return (
-        <div className="footer-container">
-            <AddIdeaButton UserIcon={ IoIosAddCircleOutline } text="Add Idea" trigger={props.trigger} setIdeaId={props.setIdeaId}/>
-        </div>
-
-    );
+        <>
```
Comment (helher) [inline src/components/footer/Footer.js:8]: Are we supposed to have an empty component? 🐧

File: src/components/sections/Section.js [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+        } catch (error) {
+            // Error can be caused by lack of Internet connection
+            alert(`Error! ${error.message}`)
+            return false;
+        }
+    }
+
+    function Colors(props) {
+        return (
+            <div className="colors" style={{backgroundColor: props.color}}>{props.children}</div>
```
Comment (helher) [inline src/components/sections/Section.js:27]: Is it not possible to style in a CSS file? I am just curious 🧐  I have not seen this before - but sure is interesting!

**Review by helher — APPROVED** [review]
Great job, I must say ☀️ I have added a few comments, but essentially it is a "go" from me ✌🏼

**Review by MLMeyer — APPROVED** [review]

**Review by szakacsliza — APPROVED** [review]

**Pull request merged**